packages/shared/components/FormLogin/FormLogin.tsx

```diff
@@ -145,28 +150,31 @@ export default function LoginForm(props: Props) {
                 onChange={e => setUser(e.target.value)}
                 placeholder="Username"
               />
-              <FieldInput
-                rule={requiredField('Password is required')}
-                label="Password"
-                value={pass}
-                onChange={e => setPass(e.target.value)}
-                type="password"
-                placeholder="Password"
-                mb={recoveryEnabled ? 0 : 4}
-              />
-              {recoveryEnabled && (
-                <ForgotLink
-                  style={{
-                    textAlign: 'right',
-                  }}
-                  to={getRecoveryStartRoute('password')}
-                >
-                  Forgot Password?
-                </ForgotLink>
-              )}
+              <Flex flexDirection="column" alignItems="flex-end" mb={3}>
```

Comment:
Unless we need to flex items, opt for `Box`

Reply:
I am using flex to have the "Forgot Password" link align to the right side